packages/components/src/Text/Text.tsx

```diff
@@ -102,15 +84,7 @@ const TypographyComponent = styled.span<BaseTypographyProps>(
   ]
 );
 
-export type TypographyProps = {
-  /**
-   * This prop can be used to specify which size text element should
-   * actually be rendered, in the case that you want to render an element
-   * as one heading but style it as if it were another. If both an `as` prop
-   * and a `size` prop are passed, the `as` will be used to determine the element
-   * and the `size` will be used to determine the styling.
-   */
-  as?: TypographyElement;
+export type TextProps = {
```

Comment:
Nit, can just call this `Props`. Especially if we don't need to export this for anything other than storybook (since we can infer the props in the stories and remove the export).